index.js

```diff
@@ -116,35 +117,41 @@
     //
     // Returns tip or direction
     tip.direction = function(v) {
-      if (!arguments.length) return direction
-      direction = v == null ? v : d3.functor(v)
+      if (!arguments.length) {
+      	return direction;
+      }
+      direction = v == null ? v : functor(v);
 
-      return tip
-    }
+      return tip;
+    };
```

Comment:
This seems to be a personal formatting preference.  Could you collapse this like the previous code?